.eslintrc.js

```diff
@@ -0,0 +1,76 @@
+module.exports = {
+    extends: ['airbnb', 'prettier', 'prettier/flowtype', 'prettier/react'],
```

Comment:
Why does this need to be so complicated?? Why do keep having airbnb there? Again, airbnb makes a lot of style/formatting assumotions.
Why not:

1- Come from eslint recommended
2- turn off what ever doesnt make sense
3- extend prettier and prettier/react?